ui-participant/src/providers/UserProvider.tsx

```diff
@@ -148,15 +148,15 @@ export default function UserProvider({ children }: { children: React.ReactNode }
     const oauthAccessToken = localStorage.getItem(OAUTH_ACCRESS_TOKEN_KEY)
     const internalLogintoken = localStorage.getItem(INTERNAL_LOGIN_TOKEN_KEY)
     if (oauthAccessToken) {
-      Api.refreshLogin(oauthAccessToken).then(loginResult => {
+      return Api.refreshLogin(oauthAccessToken).then(loginResult => {
```

Comment:
I have so much beef with async/promises 😡

Reply:
ahh, good find.  Instead of return, let's refactor this to use await and try/catch.  You're right that both are confusing, so let's stick to `async/await` as much as possible